config/jobs/kubernetes/sig-node/dra.jinja

```diff
@@ -71,7 +71,7 @@
     {%- endif %}
     spec:
       containers:
-      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20241230-3006692a6f-master
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20250121-4aed057712-master
```

Comment:
Theoretically this change should conflict with [master](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-node/dra.jinja#L77), but it doesn't for some reason. Any idea why?

Reply:
Because its bumping to the exact same version that is also used by master? `git merge` is smart enough to handle such cases.

Reply:
I could rebase to make this diff go away, but prefer not to. This won't occur in the future once https://github.com/kubernetes/test-infra/pull/34235 keeps dra.jinja up-to-date.

Adding a new job can still be problematic if the PR with the new job goes stale and the image gets bumped in the meantime.